packages/template-ui/src/views/Home.vue

```diff
@@ -129,7 +130,19 @@ export default {
       else {
         // FIXME add API to query random content nodes
         console.debug(`Fetch ${this.carouselSlideNumber} random content nodes`);
-        this.loadingCarouselNodes = false;
+
+        window.kolibri.getContentByFilter({
+          extra: {
+            channel_id: this.channel.id,
+            random: Math.random().toString(),
```

Comment:
I would prefer to ask "give me N random elements" than to passing a random number from the frontend to the backend. The backend should be a black box for the frontend.